.github/workflows/ci-ubuntu.yml

```diff
@@ -73,13 +73,19 @@ jobs:
           ${{ env.CABAL_INSTALL }}
           cd ..
 
-     # Download and test stdlib
+      # Download and test stdlib
       - name: Checkout stdlib
         uses: actions/checkout@v2
 
-      - name: Test stdlib
+      - name: Install agda-stdlib-utils
```

Comment:
Why do you need to install `agda-stdlib-utils` when you use `cabal run GenerateEverything`?  This builds the executable before running it...

Reply:
Sorry I should have been clearer: `runtests` depends on `System.Clock`
and unless I install the `clock` package globally I get the following
error when running `agda -c runtests.agda`

> Could not find module ‘System.Clock’

[cf. this log](https://github.com/agda/agda-stdlib/runs/2808733510#step:10:565)

My solution was to add `clock` as a dependency to the utils & to use
`v1-install` to force the installation of the dependencies.